sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/main/java/module-info.java

```diff
@@ -0,0 +1,5 @@
+module com.azure.messaging.eventhubs.checkpointstore.redis {
+    requires transitive com.azure.core;
+    requires redis.clients.jedis;
+    requires com.azure.messaging.eventhubs;
```

Comment:
We should also `export` all public packages in this library.

`exports com.azure.messaging.eventhubs.checkpointstore.redis`

Reply:
Should I be making this export statement from the file BlobCheckpointStore.java? The implementation is still under construction which is why I was not sure it should be exported or not. Any advice is appreciated!